indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -283,6 +168,123 @@
     }
 }
 
+.menu-entries .menu-entry,
+.reglist-filter {
```

Comment:
I still think this is overkill and rather dirty. You aren't re-using *that* much of it for the filter dropdowns.